### ISSUE TITLE
Perf dashboard: Use chrome-infra-auth to check if chromium accounts belong to Googlers

### ISSUE THREAD

**Issue by anniesullie** [catapult-project/catapult#2042]
The perf dashboard needs user accounts for two things:
   1. Users generally file bugs with their chromium.org accounts
   2. The dashboard checks if users can see internal-only data by checking if they are logged in with a google.com account

After the switch to monorail, the logged-in-as-google-posting-bugs-as-chromium-user workflow is broken, but it turns out there's now a better way to do this: we can ask users to always log in with their chromium accounts and query chrome-infra-auth to see if the chromium account is a Googler which can access internal data: https://chrome-infra-auth.appspot.com/auth/groups#external/project-chromium-googlers

@qyearsley can you work on it this week?
@chrisphn FYI

**Comment by qyearsley** [comment]
What do you think about the other alternative: always use the service account to post bugs?

One potential issue: using the chrome-infra-auth API to check whether users are internal-only will basically mean that we will need to send a request to that API for every front-end request to the perf dashboard, which might be a lot of requests. But, maybe this isn't a problem, and if it is, we could potentially cache a list of internal users (this would have to be periodically cleared).

**Comment by anniesullie** [comment]
It's pretty hard to follow the bugs that are filed from the service account, the username is quite large, and it can be hard to understand who actually filed the bug (we're planning on working on an automated follow-up mail for bug filers, for example).

We should be able to cache the list for at least a day, so I don't think the API request should be a huge problem.

**Comment by qyearsley** [comment]
Makes sense to me!

Another point  -- the list of admins for the chromeperf project (https://console.developers.google.com/permissions/projectpermissions?project=chromeperf) contains a list of @google.com accounts now, which means that we'll need to switch to google.com account whenever using admin pages, unless we change the list to contain our chromium.org accounts.

**Comment by qyearsley** [comment]
Done, deployed
